.github/workflows/deploy.yaml

```diff
@@ -40,6 +40,54 @@ jobs:
           registryUrl: https://marketplace.visualstudio.com
           extensionFile: ${{ steps.publishToOpenVSX.outputs.vsixPath }}
 
+  push-cursorless-nvim:
+    name: Push compiled files to talon.nvim plugin
+    runs-on: ubuntu-latest
+    environment: production
+    permissions:
+      deployments: write
+      contents: write
+      statuses: write
+      actions: write
+      checks: read
+    steps:
+      - uses: fidgetingbits/get-github-app-pat@v1
+        id: githubAppAuth
+        with:
+          app-id: ${{ vars.BOT_ID }}
+          app-installation-id: ${{ vars.BOT_INSTALLATION_ID }}
+          app-private-key: ${{ secrets.BOT_PRIVATE_KEY }}
```

Comment:
how come you couldn't steal the bot config from the push-cursorless-talon config below? is this setup better in some way? cc/ @fidgetingbits

Reply:
tldr; no not better, makes sense to switch to your method for consistency between the two.

this was due to my not understanding exactly how you did the bot at the time. originally when I read about setting up bots on github everything was about using github app, which I did first and how I guessed you had done it. after setting one up, your code wasn't working with it. iirc in the end if you want a github app that's part of your organization to do the signing, you normally have to pay money for something (unfortunately I forget exactly what the details were). The convoluted way around that is to mint a PAT each time, which is what I did above (though iirc in the end it's still showing up as non-verified in the commit logs). 

I later realized you had actually created a new github user for your bot, which I ended up doing as well for hands-free-vim, but haven't tried switching to a workflow similar to yours after that... Mostly because I wanted to try to get the above working after spending so much time trying to figure it out. Then I got busy with other stuff, and haven't touched it

Reply:
so would it make sense to just re-use cursorless bot for this one? or would you prefer we use your bot? I don't have a strong opinion, beyond the fact that another bot is another thing to keep track of and to keep keys for in production environment etc

Reply:
I'm fine using the cursorless bot, as long as saidelike is. makes sense to keep it easier to maintain

Reply:
done